tests/std/tests/P0218R1_filesystem/test.cpp

```diff
@@ -1216,10 +1214,10 @@ void test_directory_entry() {
     for (auto&& nonexistent : nonexistentPaths) {
         cachingEntry.assign(nonexistent); // no fail
         cachingEntry.assign(nonexistent, ec);
-        EXPECT(good(ec));
+        EXPECT(bad(ec));

```

Comment:
at some point we should consider changing these to plain asserts, but that time is not today. No change requested.,